src/ant/doc/README.md

```diff
@@ -0,0 +1,39 @@
+# Antenna Rule Checker
+
+This tool checks antenna violations based on an input LEF and DEF file, a report
+will be generated to indicate violated nets. 4 APIs are provided as an interface with FastRoute to preemptively fix antenna violation as well as used for the diode insertion flow:
```

Comment:
beginning a sentence with a number is poor form and there is no point in documenting the number of interfaces to an internal tool (fastroute) that users are not aware even exists. preemptive to what?

Reply:
preemptively, meaning at globalroute using the antenna repair flow.

Reply:
I rephrased it a little